content/en/releases/platforms.md

```diff
@@ -0,0 +1,336 @@
+| mounter                 |                                       |                                       | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          |                             |                             |
+
+#### Extra files (for GCE)
+
+There are also corresponding `*.sha256` and `*.sha512` for every file below
+
+| File                        |
+| --------------------------- |
+| configure.sh                |
+| master.yaml                 |
+| node.yaml                   |
+| shutdown.sh                 |
+| windows/common.psm1         |
+| windows/configure.ps1       |
+| windows/install-ssh.psm1    |
+| windows/k8s-node-setup.psm1 |
+| windows/user-profile.psm1   |
+
+#### Tar archives
+
+There are also corresponding `*.sha256` and `*.sha512` for every file below
+
+| File                                       | darwin/amd64 | linux/386 | linux/amd64 | linux/arm64 | linux/ppc64le | linux/s390x | windows/386 | windows/amd64 | portable |
+| ------------------------------------------ | :----------: | :-------: | :---------: | :---------: | :-----------: | :---------: | :---------: | :-----------: | :------: |
+| kubernetes-client-**[system-arch]**.tar.gz |      ✅      |    ✅     |     ✅      |     ✅      |      ✅       |     ✅      |     ✅      |      ✅       |          |
+| kubernetes-node-**[system-arch]**.tar.gz   |              |           |     ✅      |     ✅      |      ✅       |     ✅      |             |      ✅       |          |
+| kubernetes-server-**[system-arch]**.tar.gz |              |           |     ✅      |     ✅      |      ✅       |     ✅      |             |               |          |
+| kubernetes-test-**[system-arch]**.tar.gz   |      ✅      |           |     ✅      |     ✅      |      ✅       |     ✅      |             |      ✅       |    ✅    |
+| kubernetes-manifests.tar.gz                |              |           |             |             |               |             |             |               |          |
+| kubernetes-test.tar.gz                     |              |           |             |             |               |             |             |               |          |
+| kubernetes-src.tar.gz                      |              |           |             |             |               |             |             |               |          |
+| kubernetes.tar.gz                          |              |           |             |             |               |             |             |               |          |
+
+#### Other (uncategorized) files
```

Comment:
We should provide links to those files, those are not coming with artifacts.

Reply:
Removing that section completely.